src/components/task-app/task-app.component.js

```diff
@@ -46,14 +46,29 @@ const TaskApp = () => {
     );
   };
 
+  const deleteAllTasks = () => {
+    setTaskList([]);
+  };
+
   return (
     <>
-      <button role='button' onClick={completeAllTasks}>
+      <button
+        style={{ cursor: 'pointer' }}
+        role='button'
+        onClick={completeAllTasks}
+      >
         Complete All
       </button>
-      <button role='button' onClick={openForm}>
+      <button style={{ cursor: 'pointer' }} role='button' onClick={openForm}>
         Add Task
       </button>
+      <button
+        style={{ cursor: 'pointer' }}
+        role='button'
```

Comment:
buttons role by default is `button`, I'd suggest dropping this from all the buttons. 

Also, you can add a CSS style for the buttons in your [main App.css](https://github.com/Joao-Matias/to-do-list/blob/main/src/App.css) file, so you can avoid adding this inline styles on them

Reply:
c3123269175d29435167546ad670f11895b45180 Changed as advised.